deb-control

```diff
@@ -0,0 +1,11 @@
+Package: machine-emulator
+Source: machine-emulator
+Version: 0.0.0
+Architecture: amd64
```

Comment:
Don't forget to give `arm64` some love ❤️

Reply:
It shouldn't be complicated if we add `arm64` to CI. But that is out of this PR scope.
You can check the debian name of your architecture locally by running: `dpkg --print-architecture` and editing the file in the meantime. But I would wait for a bit, this PR is still WIP.

Reply:
`arm64` got included. :)